cmd/platform.go

```diff
@@ -28,6 +28,7 @@ func configurePlatform(cmd *cobra.Command) {
 	flags.StringSliceP("group", "G", nil, "The name of a GitLab organization. All repositories in that group will be used.")
 	flags.StringSliceP("user", "U", nil, "The name of a user. All repositories owned by that user will be used.")
 	flags.StringSliceP("repo", "R", nil, "The name, including owner of a GitHub repository in the format \"ownerName/repoName\".")
+	flags.StringP("repo-search", "", "", "Use a repository search to find repositories to target.")
```

Comment:
The description should mention that it is only supported for GitHub (for now).

I think it would be nice to mention how the syntax/link to reference as well.

Reply:
:+1: done!